view/src/components/todo.js

```diff
@@ -137,10 +136,30 @@ class todo extends Component {
       });
   };
 
-  deleteTodoHandler(data) {
+  /**
+   * Given an event, this function updates a state (the target of the event)
+   * with a new value
+   * @param {Event} event The event that is attempting to update a state
+   */
+  handleChange = (event) => {
+    this.setState({
+      [event.target.name]: event.target.value,
+    });
+  };
+
+  /** Returns the authentication token stored in local storage */
+  getAuth = () => {
     authMiddleWare(this.props.history);
-    const authToken = localStorage.getItem("AuthToken");
-    axios.defaults.headers.common = { Authorization: `${authToken}` };
+    return localStorage.getItem("AuthToken");
+  }
+
+  /**
+   * Takes a todo object as an input and deletes the given todo
+   * object from the database
+   * @param {todo object} data 
```

Comment:
The @param type syntax is somewhat formal, so I wouldn't recommend using the braces unless you have a formally defined type for your object data.  It's totally okay to just say `@param data`.  If you want to keep the type, it should be the actual name of the type, which in this case (in the absence of a specific type interface) would be `{Object}`.